### PR TITLE
ACP2E-160: [Cloud] Shipping prices quoted in UPS doesn't match Magento Checkout page

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) is to add a note about order price declaration for FedEx and UPS, since this affect the shipping costs.

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- N/A

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- N/A

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/shipping/fedex.html
- https://docs.magento.com/user-guide/shipping/ups.html

## Links to Magento source code or PRs

_OPTIONAL. If this pull request references a file in a Magento codebase repository or a code PR, add it here._

- https://jira.corp.magento.com/browse/ACP2E-160
- https://jira.corp.magento.com/browse/MC-43222

There have been no actual code changes, we are just documenting the existing behavior according to the product owner recommendation.

## Additional information

_OPTIONAL. What other information can you provide?_

### CODE REVIEW

File: src/shipping/fedex.md [hunk 1]
```diff
@@ -147,4 +147,7 @@ Make sure to copy or write down the authentication key. You will need it to set
    ![]({% link images/images/config-sales-shipping-methods-fedex-applicable-countries.png %}){: .zoom}
    _[FedEx Applicable Countries]({% link configuration/sales/delivery-methods.md %})_
 
+{:.bs-callout-info}
+Please note that Magento always declare the full order price to FedEx when calculating shipping charges. This behavior cannot be changed.
```
Comment (dobooth) [inline src/shipping/fedex.md:151]:
```suggestion
Magento always declares the full order price to FedEx when calculating shipping charges. This behavior cannot be changed.
```

File: src/shipping/ups.md [hunk 2]
```diff
@@ -178,3 +178,6 @@ The handling fee is optional and appears as an additional charge that is added t
    [_Sales configuration - shipping origin address options_]({% link configuration/sales/shipping-settings.md %})
 
 1. Click <span class="btn">Save Config</span>.
+
+{:.bs-callout-info}
+Please note that Magento does not declare the full order price to UPS when calculating shipping charges. This behavior cannot be changed.
```
Comment (dobooth) [inline src/shipping/ups.md:183]:
```suggestion
Magento does not declare the full order price to UPS when calculating shipping charges. This behavior cannot be changed.
```

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/shipping/ups.md [hunk 3]
```diff
@@ -178,3 +178,6 @@ The handling fee is optional and appears as an additional charge that is added t
    [_Sales configuration - shipping origin address options_]({% link configuration/sales/shipping-settings.md %})
 
 1. Click <span class="btn">Save Config</span>.
+
+{:.bs-callout-info}
+Magento does not declare the full order price to UPS when calculating shipping charges. This behavior cannot be changed.
```
Comment (leslietilling) [inline src/shipping/ups.md:183]:
```suggestion
Commerce does not declare the full order price to UPS when calculating shipping charges. This behavior cannot be changed.
```

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**